src/drivers/wayland/wayland.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file wayland
+ *
+ */
+
+#ifndef WAYLAND_H
+#define WAYLAND_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../lvgl.h"
+#if LV_USE_WAYLAND
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef bool (*lv_wayland_display_close_f_t)(lv_display_t * disp);
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+bool lw_wayland_init(void);
```

Comment:
All LVGL functions should start with `lv_`
```suggestion
bool lv_wayland_init(void);
```

Reply:
Sorry,forgot change it back...will be done.